cli/tests/pcluster/cli/test_describe_cluster_instances.py

```diff
@@ -14,23 +14,14 @@ def test_helper(self, test_datadir, run_cli, assert_out_err):
         command = ["pcluster", "describe-cluster-instances", "--help"]
         run_cli(command, expect_failure=False)
 
-        assert_out_err(
-            expected_out=(test_datadir / "pcluster-help.txt").read_text().strip(),
-            expected_err="",
-        )
+        assert_out_err(expected_out=(test_datadir / "pcluster-help.txt").read_text().strip(), expected_err="")
 
     @pytest.mark.parametrize(
         "args, error_message",
         [
-            ([""], "error: the following arguments are required: --cluster-name"),
-            (
-                ["--cluster-name"],
-                "error: argument --cluster-name: expected one argument",
-            ),
-            (
-                ["--cluster-name", "cluster", "--invalid"],
-                "Invalid arguments ['--invalid']",
-            ),
+            ([""], "error: the following arguments are required: -n/--cluster-name"),
```

Comment:
In future we might add some constants to be used in all tests for common error messages.